src/main/java/com/agricraft/agricore/plant/AgriPlant.java

```diff
@@ -112,7 +112,7 @@ public AgriPlant(String id, String plant_lang_key, String seed_lang_key, String
         this.seed_items = seed_items;
         this.stages = stages;
         this.harvestStage = harvestStage;
-        this.bonemeal = bonemeal;
+        this.fertilizers = fertilizers;
```

Comment:
The problem with this implementation is that when someone adds a fertilizer, they will then have to go and modify every single crop json to accept this fertilizer, which is less than ideal.
I think a better way to tackle this is to control it from the fertilizer jsons instead.